package dev.vality.adapter.common.exception;

public class SecretAlreadyModifyException extends RuntimeException {
    public SecretAlreadyModifyException(Throwable cause) {
        super(cause);
    }
}
